docs/getting-started/create-block/author-experience.md

```diff
@@ -12,6 +12,7 @@ To use the Placeholder, wrap the `<TextControl>` component so it becomes a child
 
 ```jsx
 import { Placeholder, TextControl } from '@wordpress/components';
+import { useBlockProps } from '@wordpress/block-editor';
```

Comment:
Totally a nitpick, but I think ideally imports should be sorted alphabetically by package name.
```suggestion
import { useBlockProps } from '@wordpress/block-editor';
import { Placeholder, TextControl } from '@wordpress/components';
```